<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myTxBuilderOrSomething</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="UTXOsGroupBox">
          <property name="title">
           <string>UTXOs</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="selectUTXOsPushButton">
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="UTXOsTableWidget">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="outputsGroupBox">
          <property name="title">
           <string>Outputs</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="addOutputPushButton">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeOutputPushButton">
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="outputsListWidget">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="4">
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QDateTimeEdit" name="dateTimeEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="dateTime">
           <datetime>
            <hour>3</hour>
            <minute>40</minute>
            <second>0</second>
            <year>2033</year>
            <month>5</month>
            <day>18</day>
           </datetime>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd HH:mm zzz</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="timeSpec">
           <enum>Qt::LocalTime</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="nLockTimeTypeComboBox"/>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="locktimeLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>2000000000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="nLockTimeRadioButton">
          <property name="text">
           <string>nLockTime</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="PreventFeeSnipingRadioButton">
        <property name="text">
         <string>Prevent fee sniping</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="RBFCheckBox">
        <property name="text">
         <string>Make final</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="signAllPUshButton">
          <property name="text">
           <string>Sign all mine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="verifyPushButton">
          <property name="text">
           <string>Verify</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportPushButton">
          <property name="text">
           <string>Export</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="broadcastPushButton">
          <property name="text">
           <string>Broadcast</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1003</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTX">
    <property name="title">
     <string>TX</string>
    </property>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuWallets">
    <property name="title">
     <string>Wallets</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_from_words"/>
    <addaction name="actionLoad_from_xprv"/>
    <addaction name="actionNew_HD"/>
    <addaction name="actionNew_empty"/>
   </widget>
   <addaction name="menuTX"/>
   <addaction name="menuWallets"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionLoad_from_words">
   <property name="text">
    <string>Regenerate from words</string>
   </property>
  </action>
  <action name="actionLoad_from_xprv">
   <property name="text">
    <string>Regenerate from xprv...</string>
   </property>
  </action>
  <action name="actionNew_HD">
   <property name="text">
    <string>New HD...</string>
   </property>
  </action>
  <action name="actionNew_empty">
   <property name="text">
    <string>New empty...</string>
   </property>
  </action>
  <action name="actionWallets">
   <property name="text">
    <string>Wallets</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
